deployment/docker/build.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+# Run build process in docker container.
+
+set -ex -o pipefail
+
+CID=
+TMPDIR=$(mktemp -d)
+function docker_rm() {
+    if [[ -z "$CID" ]]; then
+        return 0
+    fi
+    if [[ -n "$LEAVE_CONTAINER" ]]; then
+        if [[ "${LEAVE_CONTAINER}" != "0" ]]; then
+            echo "Skip to clean container: ${CID}"
+            return 0
+        fi
+    fi
+    docker rm -f "$CID"
+}
+
+trap "docker_rm; rm -rf ${TMPDIR}" EXIT
+
+BUILD_ENV_PATH=${1:?"ERROR: env file is not given."}
+
+if [[ -n "${BUILD_ENV_PATH}" && ! -f "${BUILD_ENV_PATH}" ]]; then
+  echo "ERROR: Can't find the file: ${BUILD_ENV_PATH}" >&2
+  exit 1
+fi
+
+echo "COMMIT_ID=$(git rev-parse HEAD)" >> ${BUILD_ENV_PATH}
+# /tmp is memory file system on docker.
+echo "WORK_DIR=/var/tmp/rpmbuild" >> ${BUILD_ENV_PATH}
+
+# http://stackoverflow.com/questions/19331497/set-environment-variables-from-file
+set -a
+. ${BUILD_ENV_PATH}
+set +a
+
+function docker_cp() {
+  local cid=${2%:*}
+  if [[ -z $cid ]]; then
+    # container -> host
+    docker cp $1 $2
+  else
+    # host -> container. Docker 1.7 or earlier does not support.
+    docker cp $1 $2 || {
+      local path=${2#*:}
+      tar -cO $1 | docker exec -i "${cid}" bash -c "tar -xf - -C ${path}"
+    }
+  fi
+}
+
+if [[ -n "$JENKINS_HOME" ]]; then
+  # openvnet-axsh/branch1/el7
+  img_tag=$(echo "${JOB_NAME}/${BUILD_OS}" | tr '/' '.')
+  # $BUILD_CACHE_DIR/openvnet-axsh/el7/0123abcdef.tar.gz
+  build_cache_base="${BUILD_CACHE_DIR}/${BUILD_OS}/${JOB_NAME%/*}"
+else
+  img_tag="openvnet.$(git rev-parse --abbrev-ref HEAD).${BUILD_OS}"
```

Comment:
I realized that the package name is different sometime on the release of OS. I think that installing packages is beyond the responsibility for the build script. It may only be able to do is that check the command existence unless abort in earlier.